task_11.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def find_orientation(name):
+    positive_words = ['good', 'great', 'excellent', 'positive', 'incredible', 'amazing']
+    negative_words = ['bad', 'terrible', 'poor', 'negative', 'awful']
+
+    with open(name, 'r') as task11:
+        article = task11.read().lower()
+
+    words = re.findall(r'\b\w+\b', article)
```

Comment:
do you need findall here?
why don't you just search for your positive_words and negative_words in text?